### PR TITLE
Darrell

### PR DESCRIPTION
**Pull request by DMalone87**
Changes to the database that make Vendors and Marketplaces separate from Users

### CODE REVIEW

File: tradewave/models.py [hunk 1]
```diff
@@ -23,44 +23,71 @@ class UserProperty(models.Model):
...
 
+# Entities are objects that can create and distribute credits. 
+# For now this is only markets and vendors, but this could change in the future
+class Entity(models.Model):
+    name = models.CharField(max_length=100)
+    date_created = models.DateTimeField()
+    date_active = models.DateTimeField()
+	users = ManyToManyField(User, through='Permissions') # Represents a working relationship with a User. Admin, Manager, etc.	
+	
+    class Meta:
```
Comment (nickolashe) [inline tradewave/models.py]: nice

**Comment by nickolashe** [comment]
@DMalone87 I think I see what your idea is. Let me think about this a bit please.

**Comment by permalife** [comment]
@DMalone87 please see the updates in the pull request

**Comment by DMalone87** [comment]
Looks good

**Pull request merged**